site/verifyemail.html.jet

```diff
@@ -0,0 +1,18 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("signup_page_header"), index=false)}}
+{{end}}
+
+{{block body()}}
+  <main id="main" class="page form-page">
+    <div class="page-header">
+      <h1>Verify your email</h1>
```

Comment:
All of this text needs to be translations!

Reply:
Oh whoops I completely forgot :S